.drone.yml

```diff
@@ -176,11 +176,11 @@ data:
         release_name: "auction-ui"
         values_to_set:
           - frontend.deployment.tag=$DRONE_COMMIT_SHA
-          - botTwitter.deployment.tag=$DRONE_COMMIT_SHA
+          - bot.deployment.tag=$DRONE_COMMIT_SHA
```

Comment:
Found one more place that still uses `botTwitter`:
https://github.com/sidestream-tech/unified-auctions-ui/blob/63d643212529df630479c2dbd6f95faf3364852e/kubernetes/helm/templates/aws-chamber-read-secret.yaml#L12

Reply:
Changed it 😅